ObjValMsg event
---
string    status
ObjRefMsg result
